tests/templates/kuttl/spark-connect/10-deploy-spark-connect.yaml.j2

```diff
@@ -15,7 +15,7 @@ data:
     appender.CONSOLE.filter.threshold.type = ThresholdFilter
     appender.CONSOLE.filter.threshold.level = DEBUG
 
-    rootLogger.level=INFO
+    rootLogger.level=DEBUG
```

Comment:
Do you definitely want to have DEBUG level output?

Reply:
no https://github.com/stackabletech/spark-k8s-operator/pull/567/commits/28ec1af8b80e69d114b74e1480ae935c70b43cad